### PR TITLE
Fixup server interaction

### PR DESCRIPTION
**Pull request by dwcoates**
Some changes I made while setting up pygn-mode on my new machien

### CODE REVIEW

File: pygn_server.py [hunk 1]
```diff
@@ -214,6 +214,8 @@ def generate_argparser():
 
     atexit.register(cleanup)
 
+    print(f'Server started.')
```
Comment (rolandwalker) [inline pygn_server.py:217]: Should this be to STDERR?

File: pygn_server.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (rolandwalker) [inline pygn_server.py:1]: Python 3 is not always installed as `python3`, but I'm not requesting a change.  Things are just not consistent.

**Review by rolandwalker — APPROVED** [review]

File: pygn_server.py (hunk 1 above)
Comment (dwcoates) [reply pygn_server.py:217]: It could be, but why would it be?

File: pygn_server.py (hunk 2 above)
Comment (dwcoates) [reply pygn_server.py:1]: I'll drop this. On ubuntu, `python` is explicitly python2, as far as I know.

File: pygn_server.py (hunk 1 above)
Comment (rolandwalker) [reply pygn_server.py:217]: Oh I see now that you are watching for this message at start time.  All good.

**Pull request merged**